### PR TITLE
Fix and add aeif_cond models to integration test

### PR DESCRIPTION
**Pull request by clinssen**

### CODE REVIEW

**Comment by clinssen** [comment]
See #572.

File: doc/models_library/aeif_cond_alpha.rst [hunk 1]
```diff
@@ -108,87 +103,90 @@ Source code
...
+   end
+
+   equations:
+      inline V_bounded mV = min(V_m, V_peak) # prevent exponential divergence
+      kernel g_in = (e/tau_syn_in) * t * exp(-t/tau_syn_in)
+      kernel g_ex = (e/tau_syn_ex) * t * exp(-t/tau_syn_ex)
+
+      # Add inlines to simplify the equation definition of V_m
+      inline exp_arg real = (V_bounded-V_th)/Delta_T
+      inline I_spike pA = g_L*Delta_T*exp(exp_arg)
```
Comment (pnbabu) [inline doc/models_library/aeif_cond_alpha.rst:119]:
```suggestion
      inline I_spike pA = g_L * Delta_T * exp(exp_arg)
```

File: doc/models_library/aeif_cond_exp.rst [hunk 2]
```diff
@@ -114,79 +104,82 @@ Source code
...
+   end
+
+   equations:
+      inline V_bounded mV = min(V_m, V_peak) # prevent exponential divergence
+      kernel g_in = exp(-1/tau_syn_in*t)
+      kernel g_ex = exp(-1/tau_syn_ex*t)
+
+      # Add aliases to simplify the equation definition of V_m
+      inline exp_arg real = (V_bounded-V_th)/Delta_T
+      inline I_spike pA = g_L*Delta_T*exp(exp_arg)
```
Comment (pnbabu) [inline doc/models_library/aeif_cond_exp.rst:120]:
```suggestion
      inline I_spike pA = g_L * Delta_T * exp(exp_arg)
```

File: models/aeif_cond_alpha.nestml [hunk 3]
```diff
@@ -5,24 +5,22 @@ aeif_cond_alpha - Conductance based exponential integrate-and-fire neuron model
 Description
 +++++++++++
 
-aeif_cond_alpha is the adaptive exponential integrate and fire neuron according
-to Brette and Gerstner (2005).
-Synaptic conductances are modelled as alpha-functions.
+aeif_cond_alpha is the adaptive exponential integrate and fire neuron according to Brette and Gerstner (2005). aeif_cond_exp is the adaptive exponential integrate-and-fire neuron according to Brette and Gerstner (2005), with post-synaptic conductances in the form of a bi-exponential ("alpha") function.
```
Comment (pnbabu) [inline models/aeif_cond_alpha.nestml:8]:
```suggestion
aeif_cond_alpha is the adaptive exponential integrate and fire neuron according to Brette and Gerstner (2005), with post-synaptic conductances in the form of a bi-exponential ("alpha") function.
```

**Review by pnbabu — APPROVED** [review]
Looks good to me.

**Pull request merged**